hadoop-ozone/dist/src/main/smoketest/s3/s3_serviceId.robot

```diff
@@ -0,0 +1,42 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+*** Settings ***
+Documentation       S3 gateway test with aws cli
+Library             OperatingSystem
+Library             String
+Library             BuiltIn
+Resource            ../commonlib.robot
+Resource            ./commonawslib.robot
+Test Timeout        2 minutes
+Suite Setup         Setup v4 headers
```

Comment:
`Setup v4 headers` configures AWS CLI with the secret obtained via `ozone s3 getsecret`.  This test does not use AWS CLI, rather exercises `ozone s3 getsecret` itself.

```suggestion
Documentation       Test ozone s3 getsecret command
Resource            ../commonlib.robot
Test Timeout        2 minutes
```

Also, please rename this test script to `s3_getsecret.robot`.